### PR TITLE
Lps 142507

### PR DESCRIPTION
**Pull request by Seiphon**

### CODE REVIEW

**Comment by Seiphon** [comment]
ci:test:sf

**Comment by Seiphon** [comment]
ci:test:project-templates

**Comment by Seiphon** [comment]
ci:test:relevant

**Comment by Seiphon** [comment]
ci:test:sf

**Comment by Seiphon** [comment]
ci:test:relevant

**Comment by Seiphon** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates-rest-builder/src/test/java/com/liferay/project/templates/rest/builder/ProjectTemplatesRESTBuilderWorkspaceTest.java [hunk 1]
```diff
@@ -112,6 +131,9 @@ public void testBuildTemplateRESTBuilderWorkspace() throws Exception {
 			temporaryFolder, "gradle", "gradleWS", _liferayVersion,
 			mavenExecutor);
 
+		writeGradlePropertiesInWorkspace(
+			gradleWorkspaceDir, "liferay.workspace.produc=" + _productKey);
```
Comment (petershin) [inline modules/sdk/project-templates/project-templates-rest-builder/src/test/java/com/liferay/project/templates/rest/builder/ProjectTemplatesRESTBuilderWorkspaceTest.java:135]: This looks like a typo.

**Review by petershin — CHANGES_REQUESTED** [review]

File: modules/sdk/project-templates/project-templates-rest-builder/src/test/java/com/liferay/project/templates/rest/builder/ProjectTemplatesRESTBuilderWorkspaceTest.java (hunk 1 above)
Comment (Seiphon) [reply modules/sdk/project-templates/project-templates-rest-builder/src/test/java/com/liferay/project/templates/rest/builder/ProjectTemplatesRESTBuilderWorkspaceTest.java:135]: I will drop the chages and update rest builder tool to 1.0.220, then resend the pr

**Pull request closed without merging**